external-crates/move/crates/move-compiler-transactional-tests/tests/control_flow/named_block_nesting.move

```diff
@@ -61,15 +61,15 @@ module 42::m {
     }
 
     entry fun t08() {
-        let mut i = 0;
+        let i = 0;
```

Comment:
I'm not sure whats going on here. Why did we lose this `mut`?

Reply:
Oh, is it because these `i`s are never actually observed or used due to the break, so the assignment is discarded?

Reply:
Oh actually this one seems to be a bug, or something not really fixable because of the  unused assignment. I was just blindly changing it late at night. Let me take a deeper look